### PR TITLE
ban usage of ES2015 Number props and methods

### PR DESCRIPTION
**Pull request by erwinmombay**
closes https://github.com/ampproject/amphtml/issues/3970

### CODE REVIEW

File: build-system/eslint-rules/no-es2015-number-props.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+var INVALID_PROPS = [
+  'EPSILON',
+  'isFinite',
+  'isInteger',
+  'isNaN',
+  'parseFloat',
+  'parseInt'
+];
```
Comment (aghassemi) [inline build-system/eslint-rules/no-es2015-number-props.js]: isSafeInteger,
MAX_SAFE_INTEGER,
MIN_SAFE_INTEGER

**Comment by dvoytenko** [comment]
LGTM

File: build-system/eslint-rules/no-es2015-number-props.js (hunk 1 above)
Comment (erwinmombay) [inline build-system/eslint-rules/no-es2015-number-props.js]: done.

Comment (aghassemi) [inline build-system/eslint-rules/no-es2015-number-props.js]: LGTM

**Pull request merged**